src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -252,6 +261,18 @@ public Command setLevel(double pos) {
         .withName("setLevel" + pos);
   }
 
+  public Command holdCoastMode() {
+    return startEnd(
+            () -> {
+              m_motor.setControl(new CoastOut());
```

Comment:
We should test a) if this sets both motors and b) if this works when disabled.

Reply:
On Crane it worked while disabled to toggle the motor between brake & coast mode, though those were SparkMax controllers.

Reply:
May also need to do `m_motor.stopMotor()` to abort any previous position control request, as that would continue to work even if the motor was in coast mode.